src/Layout-test-utils.js

```diff
@@ -337,11 +371,13 @@ var layoutTestUtils = (function() {
     div.style.display = 'flex';
     div.style.flexDirection = 'column';
     div.style.alignItems = 'flex-start';
+    div.style.alignContent = 'stretch';
```

Comment:
'flex-start' is similar to the previous React 'result'... but 'stretch' is the default in standard CSS so I left it at that.